easybuild/tools/systemtools.py

```diff
@@ -410,6 +410,31 @@ def get_tool_version(tool, version_option='--version'):
         return '; '.join(out.split('\n'))
 
 
+def get_gcc_version():
+    """
+    Process `gcc --version` and return the GCC version.
+    """
+    out, ec = run_cmd('gcc --version', simple=False, log_ok=False, force_in_dry_run=True, verbose=False)
+    res = None
+    if ec:
+        _log.warning("Failed to determine the version of GCC: %s" % out)
+        res = UNKNOWN
+
+    find_version = re.search("^gcc\s+\([^)]+\)\s+(?P<version>[^\s]+)\s+", out)
```

Comment:
add example of `gcc --version` output as a comment above (to help debug the regex if it's ever needed)